modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-google-cloud-vertex-ai.adoc

```diff
@@ -0,0 +1,152 @@
+image::{dir}/upload.png[files uploaded to bucket,align="center" width=75%]
+
+=== Initiating a user managed notebook instance
+
+* 1.Access Vertex AI Workbench
+
+- Return to Vertex AI Workbench in Google Cloud console.
+- Create a new User-Managed Notebook via Advanced Options or directly at https://notebook.new/.
+
+* 2. Under Details, name your notebook, select your region and select continue.
+
+image::{dir}/detailsenv.png[notebook env details,align="center" width=75%]
+
+* 3. Under Environment select **Browse** to select your startup.sh script from your Google Cloud Bucket.
+
+image::{dir}/startupscript.png[select startup script,align="center" width=75%]
+
+* 4. Select Create to initiate the notebook. It may take a few minutes for the notebook creation process to complete. When it is done, click on OPEN JUPYTERLAB. 
+
+image::{dir}/activenotebook.png[active notebook,align="center" width=75%]
+
+* On JupyterLab open a notebook with a Python kernel and run the following command for finding your notebook instance IP address.
+
+image::{dir}/python3.png[python3 kernel,align="center" width=75%]
+
+** We will whitelist this IP in your VantageCloud Lake environment in order to allow the connection.
+** This is for purposes of this guide and the notebooks demos. For production environments, a configuration of VPCs, Subnets and Security Groups might need to be configured and whitelisted.
+
+[source, python, role="content-editable"]
+----
+import requests
+def get_public_ip():
+    try:
+        response = requests.get('https://api.ipify.org')
+        return response.text
+    except requests.RequestException as e:
+        return "Error: " + str(e)
+my_public_ip = get_public_ip()
+print("My Public IP is:", my_public_ip)
+----
+
+== VantageCloud Lake Configuration
+* In the VantageCloud Lake environment, under settings, add the IP of your notebook instance
+
+image::vantagecloud-lake-demo-jupyter-sagemaker/sagemaker-lake.PNG[Initiate JupyterLab,align="center" width=75%]
+
+== Edit vars.json
+Navigate into the `lake-demos` directory in your notebook. 
+
+image::{dir}/notebooklauncher.png[notebook launcher,align="center" width=75%]
+
+Right click on vars.json to open the file with editor. 
+
+image::{dir}/openvars.png[vars.json,align="center" width=75%]
+
+Edit the *https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]* file to include the required credentials to run the demos +
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+| *"host"* 
+| Public IP value from your VantageCloud Lake environment
+| *"UES_URI"* 
+| Open Analytics from your VantageCloud Lake environment
+| *"dbc"*
+| The master password of your VantageCloud Lake environment.
+|====
+
+To retrieve a Public IP address and Open Analytics Endpoint follow these https://quickstarts.teradata.com/vantagecloud-lake/vantagecloud-lake-demo-jupyter-docker.html#_create_vantagecloud_lake_environment[instructions].
+
+====
+IMPORTANT: Change passwords in the vars.json file.
+====
```

Comment:
The callout doesn't explain to the users why they should do it. The explanation comes later. It feels out of order to me. When you ask somebody to do something, it's best to give them the reason right away.